recipes/libsodium/1.0.18/patches/libsodium_emscripten.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/dist-build/emscripten.sh b/dist-build/emscripten.sh
+index cccfca64..eb493964 100755
+--- a/dist-build/emscripten.sh
++++ b/dist-build/emscripten.sh
+@@ -64,6 +64,7 @@ echo
+ emconfigure ./configure $CONFIG_EXTRA --disable-shared --prefix="$PREFIX" \
```

Comment:
do we need patch for this? as I understand, we may specify host in `CONFIG_EXTRA`

Reply:
I tested without the patch, and the problem seems to have been fixed upstream, removing the patch.